app/domain/work_days.py

```diff
@@ -349,7 +365,9 @@ def group_user_events_by_day_with_limit(
     )
 
     if only_missions_validated_by_admin:
-        missions = [m for m in missions if not m.validated_by_admin_for(user)]
```

Comment:
:information_source: There was a bug here before. It was ok because we never used `only_missions_validated_by_admin` before.